### PR TITLE
Repose 8

### PR DESCRIPTION
**Pull request by Mario-Lopez**
My changes to get this ready for Repose 8.

### CODE REVIEW

**Comment by Mario-Lopez** [comment]
I added a Gradle build and updated the Maven build to use Repose 8.  I also removed the dependency on core-lib (now repose-core).  To test locally, you can use the second to the last commit which depends on 8.0.0.0-SNAPSHOT.  The last commit updated it to depend on 8.0.0.0.

File: README.md [hunk 1]
```diff
@@ -1,71 +1,103 @@
...
+```
 
 ## Build the cloned project
- - `cd repose-hello-world`
- - `mvn clean install`
+```
+cd repose-hello-world
+gradle clean
+gradle build
+```
```
Comment (dmnjohns) [inline README.md]: We should probably keep the documentation for Maven around if we're keeping the Maven build around.

File: README.md [hunk 2]
```diff
@@ -1,71 +1,103 @@
...
+touch ./hello-world-new/src/main/new/org/openrepose/filters/custom/helloworldnew/HelloWorldNewFilter.java
+touch ./hello-world-new/src/test/new/org/openrepose/filters/custom/helloworldnew/HelloWorldNewFilterTest.java
+```
 
-## Add the new module to the top level POM.
- - `vi ./pom.xml`
+## Add the new module to the top level build
+```
+vi ./settings.gradle
+```
```
Comment (dmnjohns) [inline README.md]: Mavens.

File: README.md [hunk 3]
```diff
@@ -1,71 +1,103 @@
...
+        'hello-world-scala',
+        'hello-world-new'
 ```
  
-## Add the new module to the custom bundle POM dependency.
- - `vi ./custom-bundle/pom.xml`
+## Add the new module to the custom bundle build
+```
+vi ./custom-bundle/build.gradle
+```
```
Comment (dmnjohns) [inline README.md]: Mavens.

File: README.md [hunk 4]
```diff
@@ -1,71 +1,103 @@
...
-        <module>hello-world-kotlin</module>
-        <module>hello-world-new</module>
-    </modules>
-    ...
+include 'custom-bundle',
+        'hello-world-groovy',
+        'hello-world-java',
+        'hello-world-kotlin',
+        'hello-world-scala',
+        'hello-world-new'
```
Comment (dmnjohns) [inline README.md]: Mavens.

File: README.md [hunk 5]
```diff
@@ -1,71 +1,103 @@
...
+apply plugin: 'ear'
+
+dependencies {
+    earlib project(":hello-world-groovy")
+    earlib project(":hello-world-java")
+    earlib project(":hello-world-kotlin")
+    earlib project(":hello-world-scala")
+    earlib project(":hello-world-new")
+}
 ```
```
Comment (dmnjohns) [inline README.md]: Mavens.

File: build.gradle [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+        provided "org.openrepose:repose-core-api"
+        provided "org.openrepose:utilities"
+
+        compile "org.jvnet.jaxb2_commons:jaxb2-basics-runtime"
+
+        testCompile "junit:junit"
+        testCompile "org.springframework:spring-test"
+        testCompile "org.mockito:mockito-all"
+        testCompile "org.apache.logging.log4j:log4j-core"
+        testCompile group: 'org.apache.logging.log4j', name: 'log4j-core', classifier: 'tests'
```
Comment (dmnjohns) [inline build.gradle]: Why is this one line using the separated syntax?

Comment (Mario-Lopez) [inline build.gradle]: Specific classifier.

Comment (dmnjohns) [inline build.gradle]: Ah, I'd guess that string notation does not support declaring the classifier.

Would it be more appropriate to either use map notation for all of these, or using string notation with a closure to set the classifier?

Comment (dmnjohns) [inline build.gradle]: I just learned that this is totally valid:
```
  testCompile(
    [group: 'junit', name: 'junit'],
    [group: 'org.springframework', name: 'spring-test']
    ...
    [group: 'org.apache.logging.log4j', name: 'log4j-core', classifier: 'tests']
  )
```

Comment (Mario-Lopez) [inline build.gradle]: I wanted to keep it consistent with Repose:
https://github.com/rackerlabs/repose/blob/repose-8-staging/repose-aggregator/core/repose-core/build.gradle#L55

Comment (dmnjohns) [inline build.gradle]: Funky.

My preference would be to keep things consistent within the same file. Whatever works, though.

**Comment by dmnjohns** [comment]
What is the reason behind declaring the `javax.ws.rs-api` dependency? I didn't see it used directly anywhere, so I'm guessing there's some other dependency that requires it?

**Comment by Mario-Lopez** [comment]
I thought it was needed when I removed core-lib, but apparently it isn't.  I removed it. 😥

**Comment by dmnjohns** [comment]
Legits.

Well this all looks good to me. Since I don't believe there's anything else to do here, I'll merge it in a few minutes.

**Pull request merged**